src/ingest_validation_tools/table-schemas/sample.yaml

```
@@ -98,6 +98,7 @@ fields:
       - Freezer (-80 Celsius)
       - Freezer (-20 Celsius)
       - Room Temperature
+      -  Refrigerator (4 Celsius)
```

```suggestion
      - Refrigerator (4 Celsius)
      - Room Temperature
```
Swapping the order, just so they are sorted correctly.